js/__internal/grids/grid_core/keyboard_navigation/m_keyboard_navigation.ts

```diff
@@ -1408,6 +1402,21 @@ export class KeyboardNavigationController extends modules.ViewController {
     });
   }
 
+  _needFocusEditingCell() {
+    const isCellEditMode = this._editingController.getEditMode() === EDIT_MODE_CELL;
+    const isBatchEditMode = this._editingController.getEditMode() === EDIT_MODE_BATCH;
+
+    const cellEditModeHasChanges = isCellEditMode && this._editingController.hasChanges();
+    const isNewRowBatchEditMode = isBatchEditMode && this._editingController.isNewRowInEditMode();
+    const $cell = this._getFocusedCell();
+
+    return (
+      !isElementDefined($cell)
```

Comment:
question
why we need focus cell if it's undefined?

Reply:
It is a technical condition. 1356 string says $cell can be null so the next (1415) string $cell.children() can throw NullReference exception.

Reply:
I mean, in this case the method will return true, while at first glance it must return false
Maybe change it to `isElementDefined($cell) && ...`?

Reply:
I checkedthis. _getFocusedCell() method, it returns $(smth) so it can never be null/false/undefined. I made this condition more simple in the next PR, thank you.